crowdsec-docs/docs/contributing/getting_started.md

```diff
@@ -0,0 +1,46 @@
+---
+id: getting_started
+title: Getting Started
+sidebar_position: 1
+---
+
+## Getting Started
+
+You want to contribute to Crowdsec? Fantastic!
+
+In order to achieve this, this guide will provide you with all the needed information and guidance :+1:
+
+### Communication
+
+The main comunication channels of the project are:
+
+- [discord](https://discord.gg/crowdsec): [Discord](https://discord.com/) is the most common communication medium of the project, and is especially suitable for live interactions (chatting).
+- [discourse](https://discourse.crowdsec.net/): [Discourse](https://www.discourse.org/) is a forum and is very useful to expose ideas and suggestions, or simply to formulate a question to which you didn't find answer on discord.
+- [github](https://github.com/crowdsecurity/): Issues and Pull Requests are used to expose bugs and suggestions in a more formal way.
+
+Do not hesitate to join & ask your questions!
+
+## Making your first contribution
+
+### Find something to work on
```

Comment:
Should we mention to go on `https://github.com/crowdsecurity/crowdsec/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22` or at least to filter issues by "good first issue" label ?